code/web/src/modules/user/Edit.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { Component } from 'react';
+import { editUser } from './api/actions'
+
+class Edit extends Component{
+  constructor(props){
+    console.log(props)
```

Comment:
There are a LOT of console.logs in here. Might want to make sure these are all taken out on the next PR